README.md

```diff
@@ -0,0 +1,46 @@
+# About
+
+Java implementation of Battleship game (client-server architecture with simple GUI in JavaFX).<br />
+See game rules: https://en.wikipedia.org/wiki/Battleship_game<br />
+One additional feature is added (Nuke - see Game features)
+
+# How to run the game?
+
+#Code quality verification:
+- Run SonarQube server on default port (9000)
+- Run **checkQuality.sh** <br />
+This script will generate Checkstyle, Findbugs, JaCoCo and Sonar reports. 
+It also opens index.html and sonar reports in Mozilla browser.
+
+Reports will be created in:
+- **/target/site/index.html** (Checkstyle, Findbugs report)
+- **/target/site/jacoco/** (directory with JaCoCo reports)
+- **/target/sonar/** (directory with sonar generated files)
+
+# Functional requirements:
+
+- One game only
+- 10x10 board
+- Fleet consists of: 4-mast ship, 2 3-mast ships, 3 2-mast ships and 4 1-mast ships.
+- Winner has ships remaining while loser has none.
+- Game messages have configurable target: console (System.out, System.err) or logs or external printer.
+- We are bi-lingual: Polish and English are fine. In future we want to add more languages: messages are to be read from a file for chosen language. Choosing the language depends on configuration variable.
+
+# Game features:
+
+- Drawing the boards for a player (fleet board has player's fleet and where opponent shot, "seen" board has where player fired and what he has shot).
```

Comment:
Better: Hit board keeps information about where and what player has already hit.